deepchecks/nlp/utils/text_properties.py

```diff
@@ -127,10 +193,33 @@ def calculate_default_properties(raw_text: Sequence[str], include_properties: Li
     default_text_properties = _get_default_properties(include_properties=include_properties,
                                                       ignore_properties=ignore_properties)
 
+    # TODO: Avoid running english specific properties if the language is not english
+    heavy_properties = [prop for prop in default_text_properties if prop['name'] in HEAVY_PROPERTIES]
+    if heavy_properties and len(raw_text) > LARGE_SAMPLE_SIZE:
```

Comment:
Maybe put all this "if" in a separate function, for code cleanliness?

Reply:
Removed the time estimation because it was 10x off